src/apb/cli.py

```diff
@@ -74,6 +74,13 @@ def subcmd_list_parser(subcmd):
         dest='basic_auth_password',
         help=u'Specify the basic auth password to be used'
     )
+    subcmd.add_argument(
```

Comment:
I think adding the `auth_token` here is not very suitable. The other subcommands, such as `apb bootstrap`, `apb remove`, and so on, also need the `auth_token` when running.

Reply:
+1, I think that `auth_token` could be added to the main list of args. In any command where interaction with the cluster is not used then the token will just be ignored.

Reply:
@jianzhangbjz , very good catch, my bad for not noticing where I put this. Now I moved to the main params.

I still can't get `apb` working locally. Would you mind giving token auth a try?

Reply:
I'm sorry I missed this message, I will have a try later today.